src/notation/view/internal/dynamicpopupmodel.cpp

```diff
@@ -0,0 +1,225 @@
+#include "dynamicpopupmodel.h"
+#include "dom/factory.h"
+#include "src/engraving/dom/dynamic.h"
+#include "types/symnames.h"
+#include "log.h"
+
+using namespace mu::notation;
+using namespace mu::engraving;
+
+static const QList<QList<DynamicPopupModel::PageItem> > DYN_POPUP_PAGES = {
+    {   // Page 1
+        { DynamicType::PP,     30, 1.5, DynamicPopupModel::Dynamic, "<sym>dynamicPiano</sym><sym>dynamicPiano</sym>" },
```

Comment:
Question: does the text need to be stored here, or would it be possible to utilise `Dynamic::dynamicText(DynamicType)`?

Reply:
It should be possible to use `Dynamic::dynamicText(DynamicType)`